src/components/smart-table-content/table-body/table-body-row/table-body-row.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import injectSheet from 'react-jss';
+import cn from 'classnames';
+import { Tr, Td, Icon } from '../../../../';
+import { colShape } from '../../shapes';
+import styles from './table-body-row.styles';
+
+class TableBodyRow extends Component {
+  state = {
+    collapsed: true,
+  };
+
+  expandRowHandler = () => {
+    this.setState(prevState => ({
+      collapsed: !prevState.collapsed,
+    }));
+  };
+
+  render() {
+    const { classes, rowData, columns, getTdContent, expandLabel } = this.props;
```

Comment:
In keeping with how our other apis and render props look, `getTdContent` perhaps should be renamed `renderTd`.